manager/state/raft/raft.go

```diff
@@ -394,6 +397,16 @@ func (n *Node) Run(ctx context.Context) error {
 				}
 			}
 
+			n.once.Do(func() {
+				// If we are the only registered member after restoring
+				// from the state, campaign to be the leader.
+				if len(n.cluster.Members()) <= 1 {
+					if err := n.Campaign(n.Ctx); err != nil {
```

Comment:
This will not work if we remove one or the other (maybe this can be refactored to use only one but `Campaign` panics on restore if done right after `RestartNode`, thus why putting it on `Run` after processing the committed entries).

It is fine campaigning twice though, as `Campaign` only triggers a step with a `MsgHup` that affects followers and candidates. In our case we are already the leader so this will be a no op.